### DISCUSSION TITLE
Remove My saved card Detail

### DISCUSSION THREAD

**Discussion by SanpreetSingh720** [community/community#109157] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Hi i have start my github account for 30 day trial period .Please remove my billing detail and card from it

**Comment by gil-son** [comment, marked as answer]
Hello,

1. Click on your picture in the top right corner and select Settings
2. Click on Billings and plans
3. Remove/Cancel the trial period

If you have problems to remove your plan:

Acces the [Github Support](https://support.github.com/)

Click on My Ticket :

![github04](https://github.com/community/community/assets/72712095/096b3477-cee6-4a19-9f1b-6fb2219b51a1)

and open a New Ticket:

![github05](https://github.com/community/community/assets/72712095/7413fc55-875a-498c-93eb-5c54d5ffc8d6)

If I supported you consider to mark my answer

God bless you!

**Comment by dopechef1** [comment]
This does not work

**Comment by arkilho** [comment]
I do have the right to ask GitHub to FORGET my credit card data.
Please tell me how to do it.

**Reply by AlejandroCanoYunes** [reply]
lol  i am looking for that sheet too

**Reply by AlejandroCanoYunes** [reply]
im trying to cancel my Copilot Individual subscription

**Reply by gexpo** [reply]
not working at all for me: requires a NEW ONE valid credit card number to substitute the old one :-(

**Reply by Lilianada** [reply]
> not working at all for me: requires a NEW ONE valid credit card number to substitute the old one :-(

This is true. I am not using any of github's paid services, kindly provide a better option for removing my card details @github

**Comment by poemihai** [comment]
This is not normal. I only made a donation for an app here and I want to remove my card. Guess a GDPR complaint at EU abilitated instituitions is in order.
